Set1/01/hello.cc

```diff
@@ -0,0 +1,8 @@
+#include <iostream>                     // importing in-out-stream
+
+using namespace std;                    // adding namespace
+
+int main()                              // args not needed explicitly
+{
+    cout << "Hello World\n";
+}
```

Comment:
Is it necessary to have these comments per the exercise guidelines? I personally think that they don't add much value, but if it's required for didactic purposes we should keep it. (There's a lot of guidelines to read through and I haven't stored them all in my head, so let me know what's the right way).

Reply:
Alright, clear to me!